src/builtins/fish_indent.rs

```diff
@@ -296,7 +296,7 @@ impl<'source, 'ast> PrettyPrinter<'source, 'ast> {
                 .binary_search(&brace_statement.source_range().start())
                 .is_err()
             {
-                for job in &brace_statement.jobs {
+                for job in &brace_statement.jobs[..] {
```

Comment:
this (a0d9c23d6ee (ast: fix MSRV breakage, 2025-09-11))
should be squashed, so that intermediate commits still build with MSRV